full-service/src/config.rs

```diff
@@ -329,7 +329,8 @@ impl LedgerDbConfig {
                     db.append_block(
                         block_data.block(),
                         block_data.contents(),
-                        block_data.signature().cloned(),
+                        block_data.signature(),
+                        block_data.metadata(),
                     )
```

Comment:
Optional/Equivalent: Call `append_block_data`

```suggestion
                    db.append_block_data(&block_data)
```